src/command_modules/azure-cli-webapp/setup.py

```diff
@@ -25,6 +25,7 @@
 
 DEPENDENCIES = [
     'azure==2.0.0rc5',
+    'azure-cli-core',
```

Comment:
support ask @lmazuel for the best practice, say, whether we need to put in a version range at the start.